pkg/manifest/os.go

```diff
@@ -152,6 +152,12 @@ type OS struct {
 	// OSTreeParent source spec (optional). If nil the new commit (if
 	// applicable) will have no parent
 	OSTreeParent *ostree.SourceSpec
+
+	// Enabling Bootupd runs bootupctl generate-update-metadata in the tree to
+	// transform /usr/lib/ostree-boot into a bootupd-compatible update
+	// payload. Only works with ostree-based images.
```

Comment:
It says `Only works with ostree-based images` - should we enforce this in code and error `if p.OStreeRef == "" && p.Bootupd`? it seems right now we just silently ignore the option? (again, honest question, maybe we never validate like this and then it would be weird to do it here only).

Reply:
Good idea.